prep-javascript-variables/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'Norman Silvera Jr';
+    console.log(fullName);
+
+    var numberOfPets = 0;
+    console.log(numberOfPets);
+
+    var likesVegetables = true;
+    console.log(likesVegetables);
+
+    var nothing = null;
+    console.log(nothing);
+
+    var never = "";
```

Comment:
You can leave off the `= ""`

```js
var never;
```

Reply:
![Screen Shot 2021-06-15 at 3 16 35 PM](https://user-images.githubusercontent.com/85140738/122130659-d5ed3280-cdec-11eb-9e1c-756623613773.png)